src/components/Kv/KvMap.vue

```diff
@@ -12,6 +12,9 @@
 <script>
 import { createIntersectionObserver } from '@/util/observerUtils';
 import testDelayedGlobalLibrary from '@/util/timeoutUtils';
+import countriesBorders from '@/assets/data/components/LenderProfile/countries-borders.json';
+import { getIntervals, mapColors } from '@/components/LenderProfile/LenderMap';
+import kvTokensPrimitives from '~/@kiva/kv-tokens/primitives.json';
```

Comment:
I think everything in this file should be in https://github.com/kiva/kv-ui-elements/blob/main/%40kiva/kv-components/vue/KvMap.vue instead of UI. Havint the country borders json file would be helpful at a global context roo.

Reply:
The related intervals util could be moved as well and unit tested in the library.

Reply:
In other words KvMap has been migrated to kv-ui-elements and all enhancements and additions should go there, then that file imported for use in the page instead of a UI only file.

Reply:
Agreed, but the suggestion in the ticket is to use this UI file: https://kiva.atlassian.net/browse/MP-415